test/lib/util/package_clause_generator_test.rb

```diff
@@ -20,7 +20,9 @@ def setup
         rpm.save!
       end
 
-      @content_view = FactoryGirl.build(:katello_content_view, :organization => organization, :repositories => [@repo])
+      @content_view = FactoryGirl.build(:katello_content_view, :organization => organization)
+      @content_view.save!
+      @content_view.repositories << @repo
```

Comment:
@ehelms @johnpmitsch We probably don't need to persist the object in the db. I made this change. Let me modify and see if the test still passes.